letsencrypt/storage.py

```diff
@@ -588,6 +612,7 @@ def new_lineage(cls, lineagename, cert, privkey, chain,
                   cli_config.live_dir):
             if not os.path.exists(i):
                 os.makedirs(i, 0700)
+                logger.debug("Creating CLI config directory %s.", i)
```

Comment:
What variable name do you think is more suitable? It would be nice to refactor the code a bit along with the logging.